utils/manifest-utils/tests/certificate_store.rs

```diff
@@ -0,0 +1,126 @@
+mod common;
+
+#[macro_use]
+extern crate serial_test;
+
+#[cfg(test)]
```

Comment:
I think there is convention that in rust integration tests there shall be no `#[cfg(test)]` and `mod tests`. You don't have it in `certificate_validation`